### ISSUE TITLE
RVIZ not installed or working in `start_gui_tools`

### ISSUE THREAD

**Issue by liampaull** [duckietown/duckietown-shell-commands#34]
(a) it's not installed - which it should be
(b) when it is installed and run it crashes:
```
root@couguar:/home/software# rviz
[ INFO] [1538772857.891672267]: rviz version 1.12.16
[ INFO] [1538772857.891738956]: compiled against Qt version 5.5.1
[ INFO] [1538772857.891804578]: compiled against OGRE version 1.9.0 (Ghadamon)
[ INFO] [1538772858.603601473]: Stereo is NOT SUPPORTED
[ INFO] [1538772858.603688698]: OpenGl version: 3 (GLSL 1.3).
Segmentation fault (core dumped)
```

**Comment by liampaull** [comment]
related? http://wiki.ros.org/docker/Tutorials/GUI

**Comment by breandan** [comment]
Should be fixed in duckietown/rpi-gui-tools@ccfdc1c. Please use duckietown/rpi-gui-tools#1 if you re-encounter this issue.
